modules/nuxt-swaggerjsonapi.yml

```diff
@@ -0,0 +1,14 @@
+name: nuxt-swaggerjsonapi
+description: 'Efficient development, fast copy and paste use for Nuxt.js'
+repo: 516310460/nuxt-swaggerjsonapi
```

Comment:
repo is : 516310460/SwarggerJSON-API on Github

Reply:
Change the repurchase address to https://github.com/516310460/nuxt-swaggerjsonapi